executor/point_get.go

```diff
@@ -339,6 +339,10 @@ func (e *PointGetExecutor) get(ctx context.Context, key kv.Key) ([]byte, error)
 func EncodeUniqueIndexKey(ctx sessionctx.Context, tblInfo *model.TableInfo, idxInfo *model.IndexInfo, idxVals []types.Datum, tID int64) (_ []byte, err error) {
 	sc := ctx.GetSessionVars().StmtCtx
 	for i := range idxVals {
+		if idxVals[i].IsNull() {
```

Comment:
I think is no the right place to fix the bug.
The idxVals should be skipped in higher level.

Reply:
We should let the caller know the built key is not valid.

Reply:
Done